crates/next-core/src/app_structure.rs

```diff
@@ -1075,10 +1075,6 @@ async fn directory_tree_to_loader_tree_internal(
                 global_metadata: global_metadata.to_resolved().await?,
             },
         );
-
-        if current_level_is_parallel_route {
-            tree.segment = rcstr!("page$");
-        }
```

Comment:
this appeared to just be leaking an internal value into the loader tree but wasn't being used by anything, so I've removed it